test/parallel/test-global-setters.js

```diff
@@ -1,4 +1,4 @@
-/* eslint-disable strict */
+'use strict';
```

Comment:
Nit: since you're making changes here, adding some code comments explaining what this test is asserting for would be helpful as a general improvement.

Reply:
That's a very good suggestion. Done.